docroot/themes/humsci/su_humsci_theme/scss/custom/archaeology/archaeology.scss

```diff
@@ -62,3 +68,8 @@ h2#menu-blockmain-menu {
     @include margin(0 null null);
   }
 }
+
+// Homepage
+.front .hs-basic-page.field-hs-page-hero {
+  @include margin(null null 0.5em);
```

Comment:
This should be in css/nodes/hs_basic_page.css file

Reply:
Are we 100% convinced that we want this to be on all homepage banners for all sites that we build with them? A paragraph with less top margin (on Archaeology they have a <p> tag that has top margin) might not come right after the hero.

Reply:
good enough reason for me 👍